js/foam/apps/calc/CalcView.js

```diff
@@ -160,8 +161,10 @@ CLASS({
         const history = Array(historyNodeList.length).fill(0).map((_,i) => historyNodeList[i])
 
         // remove the equals speech label from first history elem
-        if (history[0])
-          history[0].setAttribute('aria-label', history[0].getAttribute('aria-label').replace(/^=/,''))
+        const equals = window.chrome.i18n ? window.chrome.i18n.getMessage('Calc_ActionSpeechLabel_equals') : 'equals';
+
+        if (history[0] && history[0].getAttribute('aria-label'))
+          history[0].setAttribute('aria-label', history[0].getAttribute('aria-label').replace(new RegExp('^'+equals),''))
```

Comment:
Please put this on multiple lines (80 chars), and I think it'd be easier to read with braces.

Reply:
See https://google.github.io/styleguide/jsguide.html#formatting-column-limit for context.

Reply:
Thank you :D